cmd/zoekt-sourcegraph-indexserver/sg.go

```diff
@@ -362,6 +461,61 @@ type sourcegraphFake struct {
 	Log     *log.Logger
 }
 
+// GetRepoRank expects a file with exactly 1 line containing a comma separated
+// list of float64 as ranks.
+func (sf sourcegraphFake) GetRepoRank(ctx context.Context, repoName string) ([]float64, error) {
+	dir := filepath.Join(sf.RootDir, filepath.FromSlash(repoName))
+
+	b, err := os.ReadFile(filepath.Join(dir, "SG_REPO_RANKS"))
+	if err != nil {
+		return nil, err
+	}
+
+	return floats64(string(b)), nil
+}
+
+// GetDocumentRanks expects a file where each line has the following format:
+// path<tab>f... . where f=1-rank is a float64 score in the interval [0,1].
```

Comment:
aah, I see. I was expecting a score of 1 to mean you are very important. But I see here 0 means most important.

Reply:
I wonder if you should adjust what you return in sourcegraph internal to match more what zoekt would expect. That would also make it easier to roll out changes since if the internals of ranking changes in sourcegraph, that same PR can also update the API that zoekt uses.

Reply:
In the meantime we have updated Sourcegraph to return ranks in [0,1], with higher ranks being more important.